common/changes/@itwin/presentation-components/Presentation-FixDuplicatePropertyLabels_2022-09-27-08-51.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-components",
+      "comment": "Added property renderer",
```

Comment:
I think this message can be improved to something like this `Improved instance filter builder property renderer to avoid duplicate entries`